session/src/test/java/org/apache/iotdb/session/IoTDBSessionVectorIT.java

```
@@ -0,0 +1,219 @@
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+
+/** use session interface to IT for vector timeseries insert and select Black-box Testing */
+public class IoTDBSessionVectorIT {
+  private static final String ROOT_SG1_D1_VECTOR1 = "root.sg_1.d1.vector";
+  private static final String ROOT_SG1_D1 = "root.sg_1.d1";
+  private static final String ROOT_SG1_D2 = "root.sg_1.d2";
+
+  private Session session;
+
+  @Before
+  public void setUp() throws Exception {
+    System.setProperty(IoTDBConstant.IOTDB_CONF, "src/test/resources/");
+    EnvironmentUtils.closeStatMonitor();
+    TSFileDescriptor.getInstance().getConfig().setMaxDegreeOfIndexNode(3);
+    EnvironmentUtils.envSetUp();
+    session = new Session("127.0.0.1", 6667, "root", "root");
+    session.open();
+  }
+
+  @After
+  public void tearDown() throws Exception {
+    session.close();
+    EnvironmentUtils.cleanEnv();
+  }
+
+  @Test
+  public void alignedTest() {
+    try {
+      insertTabletWithAlignedTimeseriesMethod();
+      session.executeNonQueryStatement("flush");
+      SessionDataSet dataSet = selectTest("select * from root.sg_1.d1");
+      assertEquals(dataSet.getColumnNames().size(), 3);
+      assertEquals(dataSet.getColumnNames().get(0), "Time");
+      assertEquals(dataSet.getColumnNames().get(1), ROOT_SG1_D1_VECTOR1 + ".s1");
+      assertEquals(dataSet.getColumnNames().get(2), ROOT_SG1_D1_VECTOR1 + ".s2");
+      int time = 0;
+      while (dataSet.hasNext()) {
+        RowRecord rowRecord = dataSet.next();
+        assertEquals(rowRecord.getFields().get(0).getIntV(), time);
+        assertEquals(rowRecord.getFields().get(1).getIntV(), (time + 1) * 10 + 1);
+        assertEquals(rowRecord.getFields().get(2).getIntV(), (time + 1) * 10 + 2);
+        System.out.println(dataSet.next());
+      }
+
+      dataSet.closeOperationHandle();
+    } catch (IoTDBConnectionException e) {
+      e.printStackTrace();
```

Let the test fail if an exception is thrown.